### PR TITLE
INTYGFV-16519: Added earliest limit to date range

### PR DESCRIPTION
**Pull request by mikaelaromild1**

### CODE REVIEW

File: certificate-service/domain/src/test/java/se/inera/intyg/certificateservice/domain/validation/model/ElementValidationDateRangeListTest.java [hunk 1]
```diff
@@ -930,6 +1024,99 @@ void shouldReturnNoValidationErrorsIfCorrectPeriodsWithNoOverlap() {
 
       assertEquals(Collections.emptyList(), actualResult);
     }
+
+    @Nested
+    class BeforeMinDate {
```
Comment (ChrilleSigerhed) [inline certificate-service/domain/src/test/java/se/inera/intyg/certificateservice/domain/validation/model/ElementValidationDateRangeListTest.java:1029]: Please also add some tests to verify behaviour when min is not set.

For example, take a look at ```ElementValidationTextTest``` that handles both mandatory and limit tests.

**Review by ChrilleSigerhed — CHANGES_REQUESTED** [review]
Looks great, just need to add some tests to verify min validation is not active if not set.

File: certificate-service/app/src/main/java/se/inera/intyg/certificateservice/infrastructure/certificatemodel/fk7443/CertificateModelFactoryFK7443.java [hunk 2]
```diff
@@ -8,6 +8,7 @@
 import static se.inera.intyg.certificateservice.domain.certificatemodel.model.WorkCapacityType.TRE_FJARDEDELAR;
 
 import java.time.LocalDateTime;
+import java.time.Period;
```
Comment (mhornfeldt) [inline certificate-service/app/src/main/java/se/inera/intyg/certificateservice/infrastructure/certificatemodel/fk7443/CertificateModelFactoryFK7443.java:11]: For date validation we had min set as type TemporalAmount. Do we want to have the same here?

**Review by ChrilleSigerhed — APPROVED** [review]
Works great, good job! :)

**Pull request merged**